Basic-Recruit-Training/Person.cpp

```diff
@@ -0,0 +1,62 @@
+#include "Person.h"
+#include <iostream>
+#include <string>
+
+Person::Person()
+{
+    age_= new int(0);
+    name_= "";
+}
+
+Person::Person(const std::string& name, int age)
+{
+    age_= new int(age);
+    name_= name;
```

Comment:
Typical rule of thumb is to use initialization lists. 
`Object::Object(arg arg1)`
`: arg_(arg1)`